pages/fetcher.js

```diff
@@ -0,0 +1,3 @@
+const fetcher = (url) => fetch(url).then((res) => res.json());
```

Comment:
The `pages` folder should contain only the pages that are rendered using the NextJS router.
It would be better to move file into `src/utils`